src/generic/generic-component-breadcrumb/generic-component-breadcrumb.scss

```diff
@@ -44,4 +44,9 @@
       text-decoration: underline;
     }
   }
+
+  .ecl-breadcrumb__segment--last {
+    color: map-get($ecl-colors, 'blue-25');
+    font-weight: bold;
```

Comment:
wouldn't it make more sense if the `font-weight: bold` was applied on `.ecl-breadcrumb__segment` instead of `.ecl-breadcrumb__segment--last` + `.ecl-breadcrumb__link`?